css/style.css

```diff
@@ -0,0 +1,750 @@
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6b778c;
+  margin: 0% 0% 0% 8%;
+  list-style: disc;
+}
+
+.secondlist > :nth-child(1),
+.secondlist > :nth-child(2) {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 53px;
+  height: 24px;
+  text-align: center;
+  margin: 0% 0% 0% 3%;
+  padding-top: 5px;
+}
+
+.secondlist > :nth-child(3) {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 81px;
+  height: 24px;
+  text-align: center;
+  margin: 0% 0% 0% 3%;
+  padding-top: 5px;
+}
+
+.secondlist1 > :nth-child(1),
+.secondlist1 > :nth-child(3) {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 53px;
+  height: 24px;
+  text-align: center;
+  margin: 0 0 0 3%;
+  padding-top: 5px;
+}
+
+.secondlist1 > :nth-child(2),
+.secondlist1 > :nth-child(4) {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 81px;
+  height: 24px;
+  text-align: center;
+  margin: 0% 0% 0% 3%;
+  padding-top: 5px;
+}
+
+.skills > :nth-child(1) {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 2em;
+  letter-spacing: 0.03em;
+  color: #253858;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 98px;
+  height: 98px;
+  text-align: center;
+  padding-top: 7px;
+}
+
+.skills > :nth-child(2) {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 2em;
+  letter-spacing: 0.03em;
+  color: #253858;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 98px;
+  height: 98px;
+  text-align: center;
+  padding-top: 7px;
+}
+
+.skills > :nth-child(3) {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 2em;
+  letter-spacing: 0.03em;
+  color: #253858;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 98px;
+  height: 98px;
+  text-align: center;
+  padding-top: 7px;
+}
+
+.navbar > ul > li > a :hover {
+  color: red;
+  background: #fefefe;
+}
+
+.section1,
+.section2,
+.section3,
+.section4 {
+  display: flex;
+  width: 90%;
+  margin: auto;
+  left: 0;
+  right: 0;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+  align-items: center;
+  justify-content: space-around;
+  height: 113%;
+  padding-left: 2%;
+}
+
+.tonicpart img {
+  margin: -6% 0% 0% 60%;
+}
+
+.middle {
+  background-image: url(../img/Header-bg-desktop.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-size: 100%;
+  padding: 114px 0;
+  background-position: center;
+  background-color: #fff;
+  align-items: center;
+  margin: auto;
+}
```

Comment:
- [ ] Kindly position the background according to the design as well. 😄 
- [ ] Please give the h2 element also line-height according to the design. 😃 

## Your design

![A7](https://user-images.githubusercontent.com/103386183/199673098-2d3e7fb2-d47e-41ac-83f2-cb12924ccf2d.png)

## Figma design 

![A8](https://user-images.githubusercontent.com/103386183/199673155-6774f872-c01e-4fed-8f53-6adfd42c66e3.png)

Reply:
## issue resolved 

- thank you for noticing that error. 
- I fixed it by making another div that only contains the background and setting the size of the background to cover 
- thank you